packages/event-cache/src/backends/PostgreSQLBackend.js

```diff
@@ -2,6 +2,40 @@ const { AbstractBackend } = require('./AbstractBackend')
 
 const { debug } = require('../utils')
 
+/**
+ * Convert an undorscored_string to camelCase.
+ * @note This converts doubles (listingID) to underscored (listing_id)
```

Comment:
nit: fix comment. the function actually does the opposite.

Reply:
thanks